.github/workflows/push.yml

```
@@ -159,15 +160,21 @@ jobs:
   iOS-Build:
     if: ${{ github.actor != 'dependabot[bot]' }}
     name: iOS Build and Relaese
+    permissions:
+      contents: write
     runs-on: macos-latest
     needs: Flutter-Testing
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)